core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -215,14 +227,16 @@ object BattleDamage {
             )
                 modifiers["Tile"] = (tileDefenceBonus * 100).toInt()
 
-            for (unique in defender.unit.getMatchingUniques("[]% Strength when defending vs [] units")) {
-                if (attacker.matchesCategory(unique.params[1]))
-                    modifiers.add("defence vs [${unique.params[1]}] ", unique.params[0].toInt())
-            }
-
-            for (unique in defender.unit.getMatchingUniques("+[]% Strength when defending")) {
-                modifiers.add("Defender Bonus", unique.params[0].toInt())
-            }
+            // Deprecated since 3.17.4
+                for (unique in defender.unit.getMatchingUniques("[]% Strength when defending vs [] units")) {
+                    if (attacker.matchesCategory(unique.params[1]))
+                        modifiers.add("defence vs [${unique.params[1]}] ", unique.params[0].toInt())
+                }
+    
+                for (unique in defender.unit.getMatchingUniques("+[]% Strength when defending")) {
+                    modifiers.add("Defender Bonus", unique.params[0].toInt())
```

Comment:
Please add uniquetypes for deprecated uniques, so modders get warning signs for them

Reply:
Good point, I'll fix this later today